manifests/kustomize/README.md

```diff
@@ -81,7 +75,25 @@ Or you can add it in one line with:
 
 ```bash
 kubectl get configmap centraldashboard-config -n kubeflow -o json | jq '.data.links |= (fromjson | .menuLinks += [{"icon": "assignment", "link": "/model-registry/", "text": "Model Registry", "type": "item"}] | tojson)' | kubectl apply -f - -n kubeflow
-````
+```
+
+#### Installing Model Registry on a Kubeflow Profile
+
+Kubeflow Central Dashboard uses [Profiles](https://www.kubeflow.org/docs/components/central-dash/profiles/) to handle user namespaces and permissions. By default, the manifests deploy the Model Registry instance in the `kubeflow` namespace, to install a compatible version of Model Registry for Kubeflow, you should deploy a separate instance of Model Registry in the profile's namespace. For that just run:
+
+```shell
+cd options/istio
```

Comment:
This is inside the `model registry` directory if I'm not mistaken, so can we provide it from the directory from the root of the manifest repo?

Reply:
This doc is inside the manifest folder, so all the instructions are relative to that path, the same applies with the rest of commands above.